### DISCUSSION TITLE
Best way to submit grammar fixes in OSCAL schemas

### DISCUSSION THREAD

**Discussion by guyzyl** [usnistgov/OSCAL#1193] [Q&A] [answered]
When going over the schemas, I've noticed some issues regarding capitalization and punctuation (mostly missing dot) in the `formal-name` and `description` parts of the definitions.

I would like to make a PR to fix these, but it's probably going to change many schema lines across multiple files.
What would be the best way I can submit these fixes and make the merger process easier for you?
- Single large PR?
- PR for each file?

Let me know what you think is best and I'll submit them.

**Comment by aj-stein-nist** [comment, marked as answer]
Whether the PR is small discrete changes or one large cohesive PR is less of an issue for us, but more about how related the impact of the change(s) are. If there is a similar clarification or typo across multiple Metaschema documents, PRing them in a single targeted PR is fine. If one Metaschema document has a variety of typos that are not immediately related to each other, but generally fall in the category of typos in the SSP model for example, also understandable to gather them in one PR. But, to reiterate: it goes back to the impact of the suggested fix.

Re the white space changes, that will require tooling enhancements re #1183 and 1184. So, understandably, let's table the white space fixes for now for reasons described in the former.

As for grammar fixes and the like, submit them and we will review them as best we can. We are not making significant Metaschema changes in the near future, so this is a good time (at the time of this writing) to submit PRs! :-)
